apps/test/unit/gamelab/GameLabSpriteTest.js

```diff
@@ -23,54 +22,6 @@ describe('GameLabSprite', function () {
     createSprite = gameLabP5.p5.createSprite.bind(gameLabP5.p5);
   });
 
-  describe('method aliases', function () {
```

Comment:
This method for testing method aliases is not compatible with how the aliases work in the new `p5.play` (we no longer wrap the functions, the alias is just assigned to the existing function body). More importantly, these method aliases now have proper test cases within the p5.play repo, so we don't need to test them again in this repo.